core/src/main/scala/akka/kafka/internal/SubSourceLogic.scala

```diff
@@ -222,14 +229,14 @@ private abstract class SubSourceLogic[K, V, Msg](
 
       pendingPartitions = pendingPartitions.tail
       partitionsInStartup += tp
-      val subSource = Source.fromGraph(
-        new SubSourceStage(tp,
-                           consumerActor,
-                           subsourceStartedCB,
-                           subsourceCancelledCB,
-                           messageBuilder = this,
-                           actorNumber)
-      )
+      val graphStage = new GraphStage[SourceShape[Msg]] {
```

Comment:
Create a names class for this.

Reply:
I cleaned this up.

https://github.com/akka/alpakka-kafka/pull/930#discussion_r340823268